settings.py

```diff
@@ -1261,6 +1261,9 @@ def get_user_url(user):
         }
     }
 }
+PERSONA_HOST = 'login.persona.org'
+PERSONA_VERIFIER_URL_PATH_FORMAT = 'https://verifier.%s/verify'
+PERSONA_INCLUDE_URL_PATH_FORMAT = 'https://%s/include.js'
```

Comment:
Let's just have two settings, PERSONA_VERIFIER_URL and PERSONA_INCLUDE_URL. That's easier to debug than using string substitution.